src/app/controllers/CartController.ts

```diff
@@ -163,7 +163,7 @@ export class CartController {
   removeItem(cartId: string, cartItemId: string, sharedToken?: string): Promise<string> {
     return new Promise((resolve, reject) => {
       let mutationString = `
-                mutation ($cartId: String!, $cartItemId: String!) {
+                mutation ($cartId: String!, $cartItemId: String!, $sharedToken: String) {
```

Comment:
@junthehacker Is this how we do optional parameters??

Reply:
Yes, it is missing the exclamation mark so it's not a required arg for a graphql query